fv3core/tests/savepoint/translate/translate_riem_solver3.py

```diff
@@ -57,3 +58,7 @@ def __init__(self, grid, namelist, stencil_factory):
             "pk3": grid.default_buffer_k_dict(),
         }
         self.stencil_factory = stencil_factory
+
+    def compute_func(self, **kwargs):
+        kwargs["last_call"] = bool(kwargs["last_call"])
+        return self.riemann_solver_3(**kwargs)
```

Comment:
Was `last_call` not passed before?

Reply:
This is a casting thing - because it's a call-time argument, I didn't put the type casting in the class. I don't know where the casting from float64 to boolean was done before.